TYPO3.Flow/Documentation/TheDefinitiveGuide/PartIII/Persistence.rst

```diff
@@ -943,6 +943,37 @@ migration file only deals with one package and then move it to the *Migrations*
 in that package. This way different packages can be mixed and still a reasonable migration
 history can be built up.
 
+Ignoring tables
+---------------
+
+For tables that are not known to the schema because they are code-generated or come from a
+different system sharing the same database, the ``flow:doctrine:migrationgenerate`` command
+will generate corresponding ``DROP TABLE`` statements.
+In this case you can use the ``--filter-expression`` flag to generate migrations only for tables
```

Comment:
Is it on purpose to have only "half a paragraph" here? If you want to have separate paragraphs, an empty line must be present, and I think you want one, because you added a linebreak…